### ISSUE TITLE
QrCode not being generated (Status: INITIALIZING -> CLOSE)

### ISSUE THREAD

**Issue by vzani** [wppconnect-team/wppconnect-server#2083]
## Description

I'm trying to use wppconnect-server on Cloud Run (GCP), but qrcode is never created and status goes from initializing to close.

## Steps to Reproduce

POST https://<host>/api/mySession/<token>/generate-token
POST https://<host>/api/mySession4/start-session using the token as Bearer Token

## Log Output

Start session output:
{
    "status": "CLOSED",
    "qrcode": null,
    "version": "2.8.1"
}

Log details

POST  201  422  77 ms  PostmanRuntime/7.42.0   ->  /api/mySession/<token?>/generate-token
POST  200  259  60 ms  PostmanRuntime/7.42.0   ->  /api/mySession/start-session
http: [mySession-browser] Using browser folder './UserDataDir/mySession'
http: [mySession-browser] Initializing browser...
http: [mySession-browser] Checking headless...
http: [mySession-browser] headless option is active, browser hidden
http: [mySession-browser] Initializing...
verbose: [mySession-client] Setting WhatsApp WEB version to 2.2308.101173x
verbose: [mySession-client] Loading WhatsApp WEB
16mverbose: [mySession-client] Page loaded
16mverbose: [mySession-client] Injecting wapi.js
16mverbose: [mySession-client] WhatsApp WEB loaded
http: [mySession-client] Session Unpaired
http: [mySession-client] Waiting page load
16mverbose: [mySession-client] wapi.js injected
http: [mySession-client] Exposing onMessage function
http: [mySession-client] Exposing onAnyMessage function
http: [mySession-client] Checking isLogged
http: [mySession-client] WhatsApp WEB version: 2.2308.1011739021
http: [mySession-client] WA-JS version: 3.14.2
http: [mySession-client] Auto close configured to 60s
http: [mySession-client] Waiting for QRCode Scan...
http: [mySession-client] Auto close remain: 60s
http: [mySession-client] Registered event 2024-11-18T15:16:34.927Z
debug: [mySession-client] Exposing onAck function
debug: [mySession-client] Exposing onActionMessage function
debug: [mySession-client] Exposing onParticipantsChanged function
debug: [mySession-client] Exposing onStateChange function
debug: [mySession-client] Exposing onChatChange function
debug: [mySession-client] Exposing onAnyStateChanged function
debug: [mySession-client] Exposing onLiveLocation function
debug: [mySession-client] Exposing onGroupNotification function
debug: [mySession-client] Exposing onNewMessage function
debug: [mySession-client] Exposing onPollResponse function
debug: [mySession-client] Exposing onReaction function
debug: [mySession-client] Exposing onStatusUpdate function
debug: [mySession-client] Exposing onRemove function
debug: [mySession-client] Exposing onPresenceChanged function
debug: [mySession-client] Exposing onThreadReady function (1 registered)
http: [mySession-client] Current state: PR (CONNECTING)
http: [mySession-client] Auto close remain: 59s

**Comment by exonos** [comment]
Same here, after last update i have this issue

**Comment by ngethe** [comment]
I am having the same issue as well

**Comment by bru95** [comment]
I've been having this problem for a long time. I hope it gets resolved.

**Comment by Pankaj-111** [comment]
The same issue occurs in all versions. It happens frequently—sometimes it works fine, but after some days, it stops generating the QR code.

**Comment by renat473** [comment]
updade version whatsapp web and project wajs

**Comment by ngethe** [comment]
Yes updating the version also worked for me ensure have the latest version of the core file version
